src/azure-cli/azure/cli/command_modules/netappfiles/_params.py

```diff
@@ -205,6 +205,20 @@ def load_subvolume_arguments(self, account_name_type, pool_name_type, volume_nam
         c.argument('subvolume_name', id_part=None)
 
 
+def load_volume_quota_rules_arguments(self, account_name_type, pool_name_type, volume_name_type):
```

Comment:
The problem is that this function is not be called in `load_arguments`

Reply:
Thanks missed that one.